tests/src/Glpi/Tests/CommonTestCase.php

```diff
@@ -104,7 +104,13 @@ protected function login($name, $password, $noauto = false) {
       Session::start();
       $_SESSION['glpi_use_mode'] = Session::NORMAL_MODE;
       $auth = new Auth();
-      $result = $auth->Login($name, $password, $noauto);
+      if (defined('GLPI_PREVER') && version_compare('9.2', rtrim(GLPI_VERSION, '-dev'), 'lt')) {
+         // GLPI 9.3 and upper has this method
+         $result = $auth->login($name, $password, $noauto, false, 'local');
+      } else {
+         // older versions use this one
+         $result = $auth->Login($name, $password, $noauto);
+      }
```

Comment:
@DIOHz0r Can you explain to me the change of the login() method ? To help me to quickly approve / coment this hunk !

Reply:
For GLPI 9.3 the login method was changed, the name now is lowercase and now you can login from multiple sources like ldap or imap, On the code of login method that source is empty by default because it's sent on the login form as an hidden input with the "local" value.

Reply:
PHP is not case sensitive for function or method names (I commented about that long ago in a GLPI issue). 

I understand that the signature changed, but I guess you analyzed how this impacts ourt unit tests. Login from LDAP source was already implemented. I guess the design changed (and this can only be good !).

So this change broke our tests, am I right ? If so, I would approve your branch.. but tests still fail :D

Reply:
The tests are still failing because commit https://github.com/glpi-project/glpi/commit/a4bca634e4d055cdd6576000d8d0f2b89b0dcd91 but this change was necessary for compatibility reasons